scripts/run-app.sh

```diff
@@ -1,55 +1,56 @@
 #!/usr/bin/env sh
 
 log() {
-  local message=$1
-  local iso_8601_timestamp=$(date +%Y-%m-%dT%H:%M:%S%z)
-  echo '{"@version": 1, "@timestamp": "'"$iso_8601_timestamp"'", "level": "INFO", "message": "'"$1"'"}'
+  message=$1
+  iso_8601_timestamp=$(date +%Y-%m-%dT%H:%M:%S%z)
+  echo '{"@version": 1, "@timestamp": "'"$iso_8601_timestamp"'", "level": "INFO", "message": "'"$message"'"}'
 }
 
 export_secrets_from_dir() {
-  local secrets_dir=$1
+  for filename in "$SECRETS_DIR"/*; do
+    filename_without_path=${filename##*/}
+    filename_uppercased=$(echo "$filename_without_path" | tr '[:lower:]' '[:upper:]');
```

Comment:
fancy `tr`

Reply:
it yelled at me because the old one didn't support accented characters etc

Reply:
add quoting?